### PR TITLE
Fix that login and signup button is disabled when requirements are't …

### PR DESCRIPTION
**Pull request by StauceLukas**
Fix that login and signup button is disabled when requirements are't met.

### CODE REVIEW

File: SuppLocals/SuppLocals/ViewModels/LoginVM.cs [hunk 1]
```diff
@@ -92,15 +95,28 @@ public string Password
...
 
         public LoginVM()
         {
-            LoginClick = new RelayCommand(o => { LogInBtnClick(); }, o => true);
+            LoginClick = new RelayCommand(async (x) =>  await LogInBtnClick(), o => ButtonState());
+           
         }
 
-        public void LogInBtnClick()
+        public async Task LogInBtnClick()
```
Comment (Rangatin) [inline SuppLocals/SuppLocals/ViewModels/LoginVM.cs:119]: If it's an async Task, the name of the method should be LogInBtnClickAsync()

File: SuppLocals/SuppLocals/ViewModels/SignupVM.cs [hunk 2]
```diff
@@ -143,10 +144,11 @@ public string Email
 
         public SignupVM()
         {
-            SignupClick = new RelayCommand(o => { SignUp_ButtonClick(); }, o => true);
+            
+            SignupClick = new RelayCommand(async (x) => await SignUp_ButtonClick(), o => ButtonState());
         }
 
-        private void SignUp_ButtonClick()
+        private async Task SignUp_ButtonClick()
```
Comment (Rangatin) [inline SuppLocals/SuppLocals/ViewModels/SignupVM.cs:151]: Same applies here.

File: SuppLocals/SuppLocals/ViewModels/LoginVM.cs (hunk 1 above)
Comment (StauceLukas) [reply SuppLocals/SuppLocals/ViewModels/LoginVM.cs:119]: Hmmm, interesting, I didn't new about that. Could you provide info why ?

Comment (Rangatin) [reply SuppLocals/SuppLocals/ViewModels/LoginVM.cs:119]: As  Stephen Toub pointed out in one article: 
"If a <...> method is Task-returning and is asynchronous in nature (as opposed to a method that is known to always execute synchronously to completion but still returns a Task for some reason), it should have an “Async” suffix. That’s the guideline. The primary goal here with the naming is to make it very obvious to a consumer of the functionality that the method being invoked will likely not complete all of its work synchronously; it of course also helps with the case where functionality is exposed with both synchronous and asynchronous methods such that you need a name difference to distinguish them.<...>"

Comment (StauceLukas) [reply SuppLocals/SuppLocals/ViewModels/LoginVM.cs:119]: Interesting. Thank you for this insight ,  I will try using it in future code !

**Pull request merged**